--- libinstpatch-1.1.5/cmake/FindGtkDoc.cmake.orig	2020-05-26 13:09:21.000000000 +0200
+++ libinstpatch-1.1.5/cmake/FindGtkDoc.cmake	2021-01-17 17:02:54.626330817 +0100
@@ -34,12 +34,6 @@
     set(GTKDOC_FOUND 0)
 endif(NOT GTKDOC_SCANGOBJ_WRAPPER)
 
-find_program(GTKDOC_MKTMPL_EXE gtkdoc-mktmpl PATH "${GLIB_PREFIX}/bin")
-if(NOT GTKDOC_MKTMPL_EXE)
-	message(STATUS "gtkdoc-mktmpl not found")
-    set(GTKDOC_FOUND 0)
-endif(NOT GTKDOC_MKTMPL_EXE)
-
 find_program(GTKDOC_MKDB_EXE gtkdoc-mkdb PATH "${GLIB_PREFIX}/bin")
 if(NOT GTKDOC_MKDB_EXE)
 	message(STATUS "gtkdoc-mkdb not found")
--- libinstpatch-1.1.5/cmake/UseGtkDoc.cmake.orig	2020-05-26 13:09:21.000000000 +0200
+++ libinstpatch-1.1.5/cmake/UseGtkDoc.cmake	2021-01-17 18:15:40.099347687 +0100
@@ -130,26 +130,6 @@
             WORKING_DIRECTORY "${_output_dir}"
             VERBATIM)
 
-        # add a command to make the templates
-        add_custom_command(
-            OUTPUT
-                "${_output_unused}"
-                "${_output_undeclared}"
-                "${_output_undocumented}"
-                "${_output_tmpl_dir}"
-                "${_output_tmpl_stamp}"
-            DEPENDS
-                "${_output_types}"
-                "${_output_signals}"
-                "${_output_sections}"
-                "${_output_overrides}"
-                ${ARG_DEPENDS}
-            COMMAND ${CMAKE_COMMAND} -E remove_directory ${_output_tmpl_dir}
-            COMMAND ${GTKDOC_MKTMPL_EXE}
-                "--module=${_doc_prefix}"
-            WORKING_DIRECTORY "${_output_dir}"
-            VERBATIM)
-
         set(_copy_xml_if_needed "")
         if(ARG_XML)
             get_filename_component(ARG_XML ${ARG_XML} ABSOLUTE)
@@ -169,10 +149,10 @@
                 "${_output_sgml_stamp}"
                 "${_default_xml_file}"
             DEPENDS
-                "${_output_tmpl_stamp}"
-                "${_output_unused}"
-                "${_output_undeclared}"
-                "${_output_undocumented}"
+                "${_output_overrides}"
+                "${_output_sections}"
+                "${_output_signals}"
+                "${_output_types}"
                 ${ARG_DEPENDS}
                 ${_remove_xml_if_needed}
             COMMAND ${CMAKE_COMMAND} -E remove_directory ${_output_xml_dir}
@@ -162,6 +162,7 @@
                 "--source-suffixes=${_doc_source_suffixes}"
                 "--output-format=xml"
                 "--main-sgml-file=${_default_xml_file}"
+            COMMAND sh -c "printf '<!ENTITY package \"${PACKAGE}\">\\n<!ENTITY package_bugreport \"\">\\n<!ENTITY package_name \"${PACKAGE})\">\\n<!ENTITY package_string \"${PACKAGE}\">\\n<!ENTITY package_tarname \"${PACKAGE}\">\\n<!ENTITY package_url \"http://www.swamiproject.org/\">\\n<!ENTITY package_version \"${VERSION}\">\\n' > ${_output_xml_dir}/gtkdocentities.ent"
             ${_copy_xml_if_needed}
             WORKING_DIRECTORY "${_output_dir}"
             VERBATIM)
@@ -200,7 +180,6 @@
             DEPENDS
                 "${_output_html_dir_stamp}"
                 "${_output_sgml_stamp}"
-                "${_output_tmpl_stamp}"
                 "${ARG_XML}"
                 ${ARG_DEPENDS}
             ${_copy_xml_if_needed}
@@ -196,7 +197,7 @@
                 ${ARG_DEPENDS}
             COMMAND ${GTKDOC_FIXXREF_EXE}
                 "--module=${_doc_prefix}"
-                "--module-dir=."
+                "--module-dir=html"
                 ${ARG_FIXXREFOPTS}
             #${_remove_xml_if_needed}
             WORKING_DIRECTORY "${_output_dir}"
--- libinstpatch-1.1.5/docs/reference/CMakeLists.txt.orig	2020-05-26 13:09:21.000000000 +0200
+++ libinstpatch-1.1.5/docs/reference/CMakeLists.txt	2021-01-17 18:01:36.473917998 +0100
@@ -40,7 +40,7 @@
     CFLAGS -I${CMAKE_SOURCE_DIR}
     LDFLAGS -L${CMAKE_CURRENT_BINARY_DIR}/../../libinstpatch -linstpatch-1.0
     LDPATH ${CMAKE_CURRENT_BINARY_DIR}/../../libinstpatch
-    DEPENDS instpatch-1.0
+    DEPENDS libinstpatch
   )
 endif ()
 
--- libinstpatch-1.1.5/cmake/GtkDocScanGObjWrapper.cmake.orig	2020-05-26 13:09:21.000000000 +0200
+++ libinstpatch-1.1.5/cmake/GtkDocScanGObjWrapper.cmake	2021-01-17 19:29:27.275363616 +0100
@@ -1,3 +1,4 @@
+include(CMakeGenericSystem)
 if(NOT APPLE)
     # We use pkg-config to fing glib et al
     find_package(PkgConfig)
